web/web/api/v1/login.py

```diff
@@ -0,0 +1,164 @@
+from flask import request, jsonify, Blueprint
+from datetime import timedelta
+from werkzeug.security import generate_password_hash
+from python_http_client.exceptions import BadRequestsError
+from marshmallow import ValidationError
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy.orm.exc import NoResultFound, MultipleResultsFound
+from flask_jwt_extended import (create_access_token, get_jwt_identity,
+                                create_refresh_token, get_jti)
+from .response_wrapper import ApiResponseWrapper
+from web.database import db
+from web.models.login import Login, LoginSchema
+from web.models.user import User, UserSchema
+from web.redis import revoked_store
+
+
+login_api_bp = Blueprint('login_api_bp', __name__)
+
+
+@login_api_bp.route('/login/<int:login_id>', methods=['GET'])
```

Comment:
I think this route and the modify route seem odd to me. The client doesn't ever get their `login_id` returned to them, only access and refresh tokens. I think we should remove the login id from the path and expect an access token and use that to deduce the user. Also, are we using this GET login route for anything?

Reply:
Nope the get/put routes are not currently being used - I was just setting it up just in case if it needed to be used later, such as in the user settings page. Would you prefer me to remove these two routes, or just change the get route so it only returns the access token?

Reply:
I think we can remove this get route. As a follow up task later, we should update the get user endpoint to accept a access token and return the user after deducing it from the token.